Includes/menu.cpp

```diff
@@ -106,13 +108,22 @@ void MenuNode::down() {
 /******************************************************************************************
                                    MenuXbe
 ******************************************************************************************/
-MenuXbe::MenuXbe(MenuNode *parent, std::string const& label, std::string const& path) :
-  MenuNode(parent, label), path(path) {
+MenuXbe::MenuXbe(MenuNode *parent, std::string const& label, std::string const& paths) :
+  MenuNode(parent, label) {
+  size_t path_start = 0;
+  size_t path_end = paths.find(PATH_DELIMITER, path_start);
```

Comment:
I'm thinking this variable should be part of the `for`-loop, no?

Reply:
Sure, it just makes for a very verbose for loop. Changed.